src/package.json

```diff
@@ -1,14 +1,25 @@
 {
+  "name": "js-lotto",
+  "version": "1.0.0",
+  "description": "italian lotto",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "lotto-game"
+  },
+  "author": "LG",
+  "license": "MIT",
   "dependencies": {
     "prompt": "^1.3.0",
     "prompt-sync": "^4.2.0",
     "prompt-sync-history": "^1.0.1",
-    "prompts": "^2.4.2"
+    "prompts": "^2.4.2",
+    "terminal-kit": "^3.0.0"
   },
   "scripts": {
     "test": "jest --coverage"
   },
   "devDependencies": {
     "jest": "^29.4.0"
   }
-}
+}
```

Comment:
Why did you put these files (`package.json` and `package-lock.json`) in the `src` directory? 
Shouldn't it be in the `root` directory?